packages/RichTextEditing-Core.package/RichTextHTMLTemplate.class/class/textStructures.st

```diff
@@ -0,0 +1,5 @@
+accessing
+textStructures
+
+	textStructures ifNil: [textStructures := self initialTextStructures].
```

Comment:
Here again, can only be set once, must be reset manually by user once the value changes

Reply:
Might be for performance reasons as I mentioned above. Not quite sure if necessary though

Reply:
This is for performance reasons. When changing the template we make extensive use of collection protocol methods, that previously required the creation of this list every time, on every call. In this list every text structure is created again. This resulted in the application of markdown template taking about ten seconds, which was not acceptable to me.

Reply:
I see. Then we should reset those on program startup, as otherwise they will stay the same once set. We wouldn't be able to change them without calling `RichTextHTMLTemplate textStructures: nil` before.